neural_speed/convert/convert_bloom.py

```diff
@@ -106,6 +106,7 @@ def main(args_in: Optional[List[str]] = None) -> None:
     fout.write(struct.pack("i", 0))
     fout.write(struct.pack("i", 0))  # n_experts
     fout.write(struct.pack("i", 0))  # n_expert_used
+    fout.write(struct.pack("i", 0)) # n_embd_head_k for gemma
```

Comment:
can we have some wiser solution instead of placeholder for each model?

Reply:
Use gguf format is a solution, However, gguf is not the latest version of the pip package, and cannot support the latest model